src/adapters/email.go

```diff
@@ -2,33 +2,28 @@ package adapters
 
 import (
 	"fmt"
+	"log"
 	"net/smtp"
 )
 
 type EmailAdapter struct {
 	Username string
-	Password string
 	Auth     smtp.Auth
 }
 
-func (ea EmailAdapter) CreateAuth() {
-	ea.Auth = smtp.PlainAuth("", ea.Username, ea.Password, "smtp.gmail.com")
-}
-
 func (ea EmailAdapter) Send(target string, rate float32) {
-	println(fmt.Sprintf("Send email from: %s; to: %s; rate: %f", ea.Username, target, rate))
-	/*
-		to := []string{target}
+	//println(fmt.Sprintf("Send email from: %s; to: %s; rate: %f", ea.Username, target, rate))
+
+	to := []string{target}
 
-		msg := []byte("To: kate.doe@example.com\r\n" +
-			"Subject: USD rate\r\n" +
-			"\r\n" +
-			"USD rate: " + fmt.Sprintf("%f", rate) + "\r\n")
+	msg := []byte("To: kate.doe@example.com\r\n" +
+		"Subject: USD rate\r\n" +
+		"\r\n" +
+		"USD rate: " + fmt.Sprintf("%f", rate) + "\r\n")
 
-		err := smtp.SendMail("smtp.gmail.com:587", ea.Auth, ea.Username, to, msg)
+	err := smtp.SendMail("smtp.gmail.com:587", ea.Auth, ea.Username, to, msg)
 
-		if err != nil {
-			log.Fatal(err)
-		}
-	*/
+	if err != nil {
+		log.Fatal(err)
```

Comment:
What will happen in log.Fatal() func?

You can read about os.Exit(1)

https://go.dev/play/p/pwfwPbQ8nnb